pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -167,7 +167,7 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 		hasCephPods := false
 		for _, cephPod := range cephPods {
 			if cephPod.Spec.NodeName == request.Name {
-				logger.Debugf("cephPod.Spec.NodeName is %q and request.Name is %q", cephPod.Spec.NodeName, request.Name)
+				logger.Debugf("cephPod NodeName is same as requested Name")
```

Comment:
@leseb may have a better suggestion for the log message. I think "requested Name" is not very clear about what the code is doing at the time. I have made a suggestion below that removes redundant info while also clarifying what the code is doing.

```suggestion
				logger.Debugf("cephPod %q found on node %q being reconciled", cephPod.Name, request.Name)
```

Reply:
We can likely remove this log message completely, the node name only changes for an internal implementation detail.

Reply:
Yes, and it will print the debug log only when Name is actually equal, so no need of printing this log.

Reply:
I'd just remove it entirely.